qa/tasks/cephfs/test_nfs.py

```diff
@@ -764,8 +764,9 @@ def update_with_invalid_values(key, value, fsal=False):
                 self.ctx.cluster.run(args=['ceph', 'nfs', 'export', 'apply',
                                            self.cluster_id, '-i', '-'],
                         stdin=json.dumps(export_block_new))
-            except CommandFailedError:
-                pass
+            except CommandFailedError as e:
+                if e.exitstatus != errno.EIO:
```

Comment:
This bit won't be required once PR #53240 is merged, yes?

Reply:
that PR is going to be closed. it is going to be resolved in the underlying code(object_format.py)

Reply:
In that case, please close the PR. anything blocking?

Reply:
no, it just completely went off my mind to do it